src/main/resources/org/openpnp/machine/reference/vision/ReferenceFiducialLocator-DefaultPipeline.xml

```diff
@@ -0,0 +1,20 @@
+<pipeline>
```

Comment:
It's not quite clear to me how this pipeline is working. It seems like it has the ability to do both template matching and blob detection? Would you mind just explaining the way this works a bit?